reference/commands/creator/export-pkg.rst

```diff
@@ -60,127 +62,37 @@ and or '--profile'.
-          name = "Hello"
-          version = "0.1"
-          ...
+The :command:`export-pkg` command let you create a package from already existing files
+in your workspace, it can be useful if you are using a build process external to Conan
+and do not want to provide it with the recipe. Nevertheless, you should take into
+account that it will generate a package and Conan won't be able to guarantee its
+reproducibility or regenerate it again. This is **not** the normal or recommended flow
+for creating Conan packages.
 
-          def source(self):
-              self.run("git clone https://github.com/memsharded/hello.git")
+Execution of this command will result in several files copied to the ``package_id``
+folder in the cache (Conan will build the graph, evaluate the requirements and
+options, and call any required method to compute the id), but there could be two
+different sources for the files:
 
-          def build(self):
-              cmake = CMake(self)
-              cmake.configure(source_folder="hello")
-              cmake.build()
+ * If the argument ``--package-folder`` is provided, Conan will just copy all the
+   contents of that folder to the package one in the cache.
+ * If no ``--package-folder`` is given, Conan will execute the method ``package()``
+   of the recipe twice: first using the ``source_folder`` and then ``build_folder``
+   as the working directory.
 
-          def package(self):
-              self.copy("*.h", dst="include", src="include")
-              self.copy("*.lib", dst="lib", keep_path=False)
 
-  First we will call :command:`conan source` to get our source code in the *src* directory, then
-  :command:`conan install` to install the requirements and generate the info files, :command:`conan build` to
-  build the package, and finally :command:`conan export-pkg` to send the binary files to a package in the
-  local cache:
+There are different scenarios where this command could look like useful:
 
-  .. code-block:: bash
-      :emphasize-lines: 4
+ - You are :ref:`working locally on a package<package_dev_flow>` and you want to
+   upload it to the cache to be able to consume it from other recipes. In this situation
+   you can use the :command:`export-pkg` command to copy the package to the cache,
+   but you could also put the :ref:`package in editable mode<editable_packages>` and
+   avoid this extra step.
 
-      $ conan source . --source-folder src
-      $ conan install . --install-folder build_x86 -s arch=x86
-      $ conan build . --build-folder build_x86 --source-folder src
-      $ conan export-pkg . Hello/0.1@user/stable --build-folder build_x86 -s arch=x86
+ - You only have precompiled binaries available, then you can use the :command:`export-pkg`
+   to create the Conan package, or you can build a working recipe to donwload and
+   package them. These scenarios are described in the documentation section
+   :ref:`How to package existing binaries <existing_binaries>`.
 
 
 .. note::
```

Comment:
I am not sure if this note still applies as we have the *graph_info.json* now cc/ @memsharded

Reply:
Yes, graph_info.json captures the profile already, so the command should be fine with it.

Reply:
So... we leave the note as it is now, don't we?

Reply:
@memsharded it is not clear to me: Is the ``--profile`` argument still needed to cover some corner cases of the export-pkg flow?

Reply:
IMHO, recommending the usage of `--profile` (or settings, options,...) is dangerous as you are going to package/copy existing files which have been created with the `settings/options` that should be in the `graph_info.json`, so if you pass other profile/settings/options you can package files that do not correspond to the profile used to generate them... (I think we are not doing any check here).